commands/scp.go

```diff
@@ -0,0 +1,136 @@
+package commands
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/codegangsta/cli"
+	"github.com/docker/machine/libmachine"
+	"github.com/docker/machine/log"
+)
+
+var (
+	ErrMalformedInput = fmt.Errorf("The input was malformed")
+)
+
+var (
+	// TODO: possibly move this to ssh package
+	baseSSHOptions = []string{
+		"-o", "IdentitiesOnly=yes",
+		"-o", "StrictHostKeyChecking=no",
+		"-o", "UserKnownHostsFile=/dev/null",
+		"-o", "LogLevel=quiet", // suppress "Warning: Permanently added '[localhost]:2022' (ECDSA) to the list of known hosts."
+	}
+)
+
+func getInfoForScpArg(hostAndPath string, mcn libmachine.Machine) (*libmachine.Host, string, []string, error) {
```

Comment:
Could you name the return values here? It'd clarify what the 4 result params are
```go
func getInfoForScpArg(hostAndPath string, mcn libmachine.Machine) (host *libmachine.Host, path string, args []string, err error) {
```